py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py

```diff
@@ -15,58 +15,57 @@
-    eta = Param(Params._dummy(), "eta", "eta")
-    learnRateAnnealing = Param(Params._dummy(), "learnRateAnnealing", "Learn Rate Annealing")
-    sampleRate = Param(Params._dummy(), "sampleRate", "Sample rate")
-    subsample = Param(Params._dummy(), "subsample", "subsample")
-    colSampleRate = Param(Params._dummy(), "colSampleRate", "col sample rate")
-    colSampleByLevel = Param(Params._dummy(), "colSampleByLevel", "Col Sample By Level")
-    colSampleRatePerTree = Param(Params._dummy(), "colSampleRatePerTree", "col samle rate")
-    colsampleBytree = Param(Params._dummy(), "colsampleBytree", "col sample by tree")
-    maxAbsLeafnodePred = Param(Params._dummy(), "maxAbsLeafnodePred", "max abs lead node prediction")
-    maxDeltaStep = Param(Params._dummy(), "maxDeltaStep", "max delta step")
-    scoreTreeInterval = Param(Params._dummy(), "scoreTreeInterval", "score tree interval")
-    initialScoreInterval = Param(Params._dummy(), "initialScoreInterval", "Initial Score Interval")
-    scoreInterval = Param(Params._dummy(), "scoreInterval", "Score Interval")
-    minSplitImprovement = Param(Params._dummy(), "minSplitImprovement", "Min split improvement")
-    gamma = Param(Params._dummy(), "gamma", "gamma")
-    nthread = Param(Params._dummy(), "nthread", "nthread")
-    maxBins = Param(Params._dummy(), "maxBins", "nbins")
-    maxLeaves = Param(Params._dummy(), "maxLeaves", "max leaves")
-    minSumHessianInLeaf = Param(Params._dummy(), "minSumHessianInLeaf", "min sum hessian in leaf")
-    minDataInLeaf = Param(Params._dummy(), "minDataInLeaf", "min data in leaf")
-    treeMethod = Param(Params._dummy(), "treeMethod", "Tree Method")
-    growPolicy = Param(Params._dummy(), "growPolicy", "Grow Policy")
-    booster = Param(Params._dummy(), "booster", "Booster")
-    dmatrixType = Param(Params._dummy(), "dmatrixType", "DMatrix type")
-    regLambda = Param(Params._dummy(), "regLambda", "req lambda")
-    regAlpha = Param(Params._dummy(), "regAlpha", "req aplha")
-    sampleType = Param(Params._dummy(), "sampleType", "Dart Sample Type")
-    normalizeType = Param(Params._dummy(), "normalizeType", "Dart Normalize Type")
-    rateDrop = Param(Params._dummy(), "rateDrop", "rate drop")
-    oneDrop = Param(Params._dummy(), "oneDrop", "onde drop")
-    skipDrop = Param(Params._dummy(), "skipDrop", "skip drop")
-    gpuId = Param(Params._dummy(), "gpuId", "GPU id")
-    backend = Param(Params._dummy(), "backend", "Backend")
+    quietMode = Param(Params._dummy(), "quietMode", "Quiet mode", H2OTypeConverters.toBoolean())
+    ntrees = Param(Params._dummy(), "ntrees", "Number of trees", H2OTypeConverters.toInt())
+    nEstimators = Param(Params._dummy(), "nEstimators", "number of estimators", H2OTypeConverters.toInt())
+    maxDepth = Param(Params._dummy(), "maxDepth", "Maximal depth", H2OTypeConverters.toInt())
+    minRows = Param(Params._dummy(), "minRows", "Min rows", H2OTypeConverters.toFloat())
```

Comment:
Any idea why this could be a float number? I would expect integers only...

Reply:
Yup, it is weird, but as H2O accepts Double values wanted to be consistent